samples/settings/boards/esp32.conf

```diff
@@ -0,0 +1,38 @@
+CONFIG_WIFI=y
```

Comment:
Adding `esp32.conf` is separate logical change from "improve shell commands to set/get setting", so please create a separate commit and preferably a separate PR for that.

Reply:
i removed from this PR and created a new one